package.json

```diff
@@ -46,18 +46,18 @@
     "@googleforcreators/story-editor": "*",
     "@jsdevtools/coverage-istanbul-loader": "^3.0.5",
     "@prettier/plugin-xml": "^1.2.0",
-    "@storybook/addon-a11y": "^6.3.6",
-    "@storybook/addon-actions": "^6.3.6",
-    "@storybook/addon-backgrounds": "^6.3.6",
-    "@storybook/addon-docs": "^6.3.6",
-    "@storybook/addon-knobs": "^6.3.0",
-    "@storybook/addon-links": "^6.3.12",
-    "@storybook/addon-storysource": "^6.3.6",
-    "@storybook/addon-viewport": "^6.3.6",
-    "@storybook/client-api": "^6.3.6",
-    "@storybook/client-logger": "^6.3.6",
+    "@storybook/addon-a11y": "^6.4.14",
+    "@storybook/addon-actions": "^6.4.14",
```

Comment:
Since `'@storybook/addon-actions', '@storybook/addon-docs','@storybook/addon-knobs',  '@storybook/addon-viewport/register', '@storybook/addon-backgrounds/register',` are all getting removed from main.cjs `addons` array can they also be removed as deps here?

Reply:
Not yet. We need to update all stories that use those add-on packages. For example there are still ~51 stories that use `actions`, which are coming from`@storybook/addon-actions`. Since those stories in particular weren't using `knobs` I figured it would be nice to spilt up those updates.  

We had to remove them from the `main.cjs` otherwise I would get a weird double instance error. From Storybook 
```
Essentials is "zero-config”. It comes with a recommended configuration out of the box.

If you need to reconfigure any of the individual essential addons, install them manually by following its installation instructions and adjusting its configuration to suit your needs.
```

Reply:
Got it, that makes sense. 

Can you go ahead and make a ticket for the actions update?

Reply:
Yes!

Reply:
#10375 created.